server.js

```diff
@@ -335,7 +335,8 @@ app.post('/questionedit', function(req, res) {
 app.get('/statistics', function(req, res) {
     if (req.session.adminQuestionList == null) {
         selector.findQuestions(0, selector.findTypes.SORT_DEFAULT,
-                                    null, function(questionlist) {
+				    //TODO Does statics pull from all questions or only questions the student can see?
```

Comment:
I think it should be only questions the student can see.

Reply:
Ok, it was unclear and was going to be asked during tommorrow's meeting.